spec/controllers/api/v1/superheroes_controller_spec.rb

```diff
@@ -2,20 +2,34 @@
 require 'spec_helper'
 
 RSpec.describe Api::V1::SuperheroesController, type: :controller do
+
+  let!(:kjoya) {
+    User.create(
+      first_name: 'kylee',
+      last_name: 'a',
+      username: 'kjoya',
+      email: 'kjoya@gmail.com',
+      password: 'secrectPassword',
+      encrypted_password: 'secretPassword'
+    )
+  }
+
   let!(:magneto) {
     Superhero.create(
       name: 'Magneto',
       superpower: 'BENDS METAL',
       backstory: 'so sad',
-      image_url: 'magneto.com'
+      image_url: 'magneto.com',
+      user: kjoya
     )
   }
   let!(:hulk) {
     Superhero.create(
       name: 'Hulk',
       superpower: 'very strong',
       backstory: 'epic',
-      image_url: 'hulk.com'
+      image_url: 'hulk.com',
+      user: kjoya
     )
   }
```

Comment:
I'm not going to have you go back and change this, but I'm curious as to why you're not using factoryBot here.  I thought you used it for other tests?